Lib/test/test_global.py

```diff
@@ -25,15 +25,15 @@ def wrong1():
     global a
     global b
 """
-        check_syntax_error(self, prog_text_1, lineno=4, offset=5)
+        check_syntax_error(self, prog_text_1, lineno=4, offset=4)
```

Comment:
We mostly try not to make changes in cpython-originated code.
Please keep this as original code and add `@unittest.expectedFailure` if you cannot keep this compatibility with the patch.

Reply:
fixed

Reply:
I think this change might be handle the same way this CPython commit handles it [025eb98dc0](https://github.com/python/cpython/commit/025eb98dc0c1dc27404df6c544fc2944e0fa9f3a): make all syntax error 1-based by incrementing location in "constructor" by 1

Reply:
Oh, thank you! good to know.